src/Resource/Resource.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Jahuty\Resource;
+
+abstract class Resource
+{
+    abstract public static function from(array $payload): Resource;
```

Comment:
: - (

Reply:
I'm a little more anal about these things, but I don't like each resource having to define how to create itself. I personally would implement something like [this](uhttps://packagist.org/packages/league/containerrl), but it's such a small library already I'm not sure how worth it it is

Reply:
Yeah, you're right. It's not great. I switched it to an interface, but that's not any better. I might leave it as is for now, since there are only two resources, though. When we start adding more, we should figure out a better way to handle it.